pmdarima/model_selection/_split.py

```diff
@@ -108,16 +110,19 @@ def split(self, y, exogenous=None):
         test : np.ndarray
             The test set indices for the split
         """
-        y, exog = indexable(y, exogenous)
+        # Temporary shim until we remove `exogenous` support completely
+        X, _ = pm_compat.get_X(X, **kwargs)
+
+        y, exog = indexable(y, X)
```

Comment:
Should this be changed to `X`?

```suggestion
        y, X = indexable(y, X)
```